real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/Robot.java

```diff
@@ -47,10 +46,19 @@ private Robot() {
 		nodeList = new LinkedList<>();
 		new RobobuggyLogicNotification("Logic Exception Setup properly" ,  RobobuggyMessageLevel.NOTE);
 		// Initialize Nodes
-		nodeList.add(new GpsNode(NodeChannel.GPS, RobobuggyConfigFile.COM_PORT_GPS));
-		nodeList.add(new ImuNode(NodeChannel.IMU, RobobuggyConfigFile.COM_PORT_IMU));
-		nodeList.add(new RBSMNode(NodeChannel.ENCODER, NodeChannel.STEERING,
-				RobobuggyConfigFile.COM_PORT_RBSM, COMMAND_PERIOD));
+//		nodeList.add(new GpsNode(NodeChannel.GPS, RobobuggyConfigFile.COM_PORT_GPS));
```

Comment:
For master I want for all nodes to stayed uncommented.  Ideally we should be able to run master on the surface from a clean clone without changing anything in the code